src/elements/shared-styles.html

```diff
@@ -44,6 +48,10 @@
         text-decoration: none;
       }
 
+      .roboto-font {
```

Comment:
This font-family is a default for this app
https://github.com/gdg-x/hoverboard/blob/hoverboard-v2/index.html#L78